### PR TITLE
Update linux-exclusions.md

### LINKED ISSUE #12168: EDR Exclusion information is not complete
The note about Defender Exclusion not applying to EDR Exclusion is useful, however from Customer perspective, it is confusing because there are no reference to EDR Exclusion, on how to do that? Currently the only method to do it is by Contacting Customer support. Please add this information so customer could directly refer to us when they need EDR exclusion.

image

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 7d6bcf39-9a8c-7fa3-803a-508e7b74cc9b
* Version Independent ID: 9fd6d652-df16-7f4b-d253-6aee2362f836
* Content: [Configure and validate exclusions for Microsoft Defender for Endpoint on Linux](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/linux-exclusions?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/linux-exclusions.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/linux-exclusions.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
Added details about contacting support for EDR Exclusions 

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12168

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

File: microsoft-365/security/defender-endpoint/linux-exclusions.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ ms.date: 12/18/2020
 This article provides information on how to define exclusions that apply to on-demand scans, and real-time protection and monitoring.
 
 > [!IMPORTANT]
-> The exclusions described in this article don't apply to other Defender for Endpoint on Linux capabilities, including endpoint detection and response (EDR). Files that you exclude using the methods described in this article can still trigger EDR alerts and other detections.
+> The exclusions described in this article don't apply to other Defender for Endpoint on Linux capabilities, including endpoint detection and response (EDR). Files that you exclude using the methods described in this article can still trigger EDR alerts and other detections. For endpoint detection and response (EDR) exclusions, [contact support](/microsoft-365/admin/get-help-support).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/linux-exclusions.md]:
```suggestion
> The exclusions described in this article don't apply to other Defender for Endpoint on Linux capabilities, including endpoint detection and response (EDR). Files that you exclude using the methods described in this article can still trigger EDR alerts and other detections. For EDR exclusions, [contact support](/microsoft-365/admin/get-help-support).
```

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**